app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def show
```

Comment:
`Show` is a keword action used on CRUD actions.

It is confusing to call a `show` method in this case...